# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ldapuserfolder/posixgroup.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./utils.py.orig	2005-05-13 15:48:51.335974584 -0400
+++ ./utils.py	2005-05-13 15:49:19.611028000 -0400
@@ -35,6 +35,7 @@
 GROUP_MEMBER_MAP = { 'groupOfUniqueNames' : 'uniqueMember'
                    , 'groupOfNames' : 'member'
                    , 'accessGroup' : 'member'
+                   , 'posixGroup' : 'member'
                    , 'group' : 'member'
                    }
 
